docs/configuration.md

```diff
@@ -184,22 +195,39 @@ The below is the default which is used if this is not configured in either Strat
 
 !!! Note
     Not all exchanges support "market" orders.
-    The following message will be shown if your exchange does not support market orders: `"Exchange <yourexchange>  does not support market orders."`
+    The following message will be shown if your exchange does not support market orders:
+    `"Exchange <yourexchange>  does not support market orders."`
 
 !!! Note
-    stoploss on exchange interval is not mandatory. Do not change it's value if you are unsure of what you are doing. For more information about how stoploss works please read [the stoploss documentation](stoploss.md).
+    Stoploss on exchange interval is not mandatory. Do not change its value if you are
+    unsure of what you are doing. For more information about how stoploss works please
+    read [the stoploss documentation](stoploss.md).
 
 ### Understand order_time_in_force
-`order_time_in_force` defines the policy by which the order is executed on the exchange. Three commonly used time in force are:<br/>
+The `order_time_in_force` configuration parameter defines the policy by which the order
+is executed on the exchange. Three commonly used time in force are:
+
 **GTC (Goog Till Canceled):**
```

Comment:
i think this should be `Good Till Cancelled` - not `Goog` ... ...

Reply:
Done, pushed.